### DISCUSSION TITLE
selective chapter title alignment in quarto book (scrbook)   continuing issue #4546

### DISCUSSION THREAD

**Discussion by nithinmkp** [quarto-dev/quarto-cli#10675] [Q&A] [answered]
### Description

I am writing on the issue #4546
 my university requires all chapter headings to be right aligned. i am not able to achieve this

**Comment by mcanouil** [comment, marked as answer]
We can't help without code.
Your question is a LaTeX question, please do search on the internet on how to do this in LaTeX.

Then you can apply what you will find the same way as in the discussion you've linked.

See also:

- https://quarto.org/docs/journals/templates.html#latex-partials
- https://quarto.org/docs/output-formats/pdf-basics.html#latex-includes

**Reply by nithinmkp** [reply]
This solved it.

````{=latex}
\renewcommand*{\chapterformat}{%
  \hfill \chapapp~\thechapter\autodot\quad
}
\renewcommand*{\chapterlineswithprefixformat}[3]{%
  \raggedleft#2\par\nobreak\raggedleft#3
}
````
I have this in my thesis.sty file
